frontend/packages/ux-editor/src/components/Properties/EditSubformTableColumns/ColumnElement/ColumnElement.test.tsx

```diff
@@ -111,6 +112,18 @@ describe('ColumnElement', () => {
 
     expect(onDeleteColumnMock).toHaveBeenCalledTimes(1);
   });
+
```

Comment:
Here maybe we can add test or update this to test the scenario where the user clicks first on the button `Legg til kolonne` and check if the edit mode is showing.